src/components/dar_collection_table/DarCollectionTable.js

```diff
@@ -27,147 +28,86 @@ const styles = {
-  const {darCode, name, submissionDate, status, actions} = columnHeaderFormat;
-  return [darCode, name, submissionDate, status, actions];
-};
-
-const projectTitleCellData = ({projectTitle = '- -', darCollectionId, style = {}, label = 'project-title'}) => {
-  return {
-    data: projectTitle,
-    id: darCollectionId,
-    style,
-    label
-  };
-};
-
-const darCodeCellData = ({darCode = '- -', darCollectionId, style = {}, label = 'dar-code'}) => {
-  return {
-    data: darCode,
-    id: darCollectionId,
-    style,
-    label
-  };
-};
-
-const submissionDateCellData = ({createDate, darCollectionId, style = {}, label = 'submission-date'}) => {
-  return {
-    data: isNil(createDate) ? '- - ' : formatDate(createDate),
-    id: darCollectionId,
-    style,
-    label
-  };
-};
-
-const statusCellData = ({status = '- -', darCollectionId, style = {}, label = 'status'}) => {
-  return {
-    data: status,
-    id: darCollectionId,
-    style,
-    label
-  };
-};
-
-const CancelCollectionButton = (props) => {
-  const { collection } = props;
-  return h(SimpleButton, {
-    keyProp: `cancel-collection-${collection.id}`,
-    label: 'Cancel',
-    disabled: includes(determineCollectionStatus(collection))(nonCancellableCollectionStatuses),
-    baseColor: Theme.palette.secondary,
-    additionalStyle: {
-      width: '30%',
-      padding: '2%',
-      fontSize: '1.45rem',
-    },
-    onClick: () => props.showConfirmationModal(collection)
-  });
-};
-
-const ResubmitCollectionButton = (props) => {
-  const { collection } = props;
-  return h(SimpleButton, {
-    keyProp: `resubmit-collection-${collection.id}`,
-    label: 'Revise',
-    baseColor: Theme.palette.secondary,
-    additionalStyle: {
-      width: '30%',
-      padding: '2%',
-      fontSize: '1.45rem',
-    },
-    onClick: () => props.showConfirmationModal(collection)
-  });
-};
-
-const actionsCellData = ({collection, showConfirmationModal}) => {
-  const { darCollectionId } = collection;
-  const cancel = {
-    isComponent: true,
-    id: darCollectionId,
-    label: 'cancel-button',
-    data: div(
-      {
-        style: {
-          display: 'flex',
-          justifyContent: 'left'
-        },
-        key: `cancel-collection-cell-${darCollectionId}`
-      },
-      [h(CancelCollectionButton, {collection, showConfirmationModal})]
-    )};
-  const revise = {
-    isComponent: true,
-    id: darCollectionId,
-    label: 'resubmit-button',
-    data: div(
-      {
-        style: {
-          display: 'flex',
-          justifyContent: 'left'
-        },
-        key: `resubmit-collection-cell-${darCollectionId}`
-      },
-      [h(ResubmitCollectionButton, {collection, showConfirmationModal})]
-    )
-  };
-
-  return isCollectionCanceled(collection) ? revise : cancel;
+  const {darCode, name, submissionDate, pi, institution, datasetCount, status, actions} = columnHeaderFormat;
+  return [darCode, name, submissionDate, pi, institution, datasetCount, status, actions];
```

Comment:
I just realized something. Not only do the actions buttons change between columns, but also the columns themselves. However that's something we can cover in another refactoring ticket (since it's less about this component specifically and more about prop data abstraction). Just leaving a note here as a point of reference.